backend/migrate/migrate.py

```diff
@@ -45,10 +45,11 @@ def create_connect_db():
             _db_name = os.getenv("DATABASE_NAME", "artemis_db")
```

Comment:
make sure you test the migration before merging (e.g., from version 11 to 12, or simply a custom version)

Reply:
it's the same command